src/styles/blocks/catalog.scss

```diff
@@ -18,6 +19,13 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: color 0.3s, transform 0.3s;
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
+    }
```

Comment:
use `SCSS` variables for values, used in several places throughout the code

Reply:
fixed

Reply:
push it, I don't see any commits

Reply:
just pushed